components/indexer.py

```diff
@@ -14,10 +14,18 @@ class Indexer:
     intake_arm_piston: wpilib.Solenoid
     intake_left_motor: wpilib.interfaces.SpeedController  # Looking from behind the robot
     intake_right_motor: wpilib.interfaces.SpeedController  # Looking from behind the robot
-    SHIMMY_TICKS = int(50 * 0.25)
+
+    indexer_speed = tunable(0.6)
+    injector_speed = tunable(0.7)
+    intake_motor_speed = tunable(1.0)
+    shimmy_speed = tunable(1.0)
+    shimmy_ticks = tunable(50 // 4)
```

Comment:
This seems less readable than the previous version

Reply:
Happy to change this back then.